spec/lucky/params_spec.cr

```diff
@@ -156,6 +165,20 @@ describe Lucky::Params do
       params.nested(:user).should eq({"name" => "Paul", "age" => "28"})
     end
 
+    it "gets nested query params" do
+      request = build_request body: "", content_type: ""
```

Comment:
Maybe do what you did with the JSON spec and add some form params here to make sure thy are merged?

Reply:
hold on... laughing to hard at that gif... 😹